packages/server/lib/middleware/access.middleware.ts

```diff
@@ -456,7 +461,8 @@ async function fillLocalsFromSession(req: Request, res: Response<any, RequestLoc
 
         res.locals['user'] = user;
 
-        if (ignoreEnvPaths.includes(req.route.path)) {
+        const fullPath = path.join(req.baseUrl, req.route.path);
```

Comment:
because we use subrouter and express is dumb we have to compute our own full path